modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -110,7 +110,8 @@ public ListTypeEntry getListTypeEntry(long listTypeDefinitionId, String key)
 	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ListTypeEntry updateListTypeEntry(
-			long listTypeEntryId, Map<Locale, String> nameMap)
+			String externalReferenceCode, long listTypeEntryId,
```

Comment:
Can you move this commit to be the second one? this way you just need to run and commit buildService once.